pagopa/pn-frontend-e2e
# PN-9865-e-2-e-fix-login-steps-che-falliscono-in-aws

Issue Description
Currently, all tests for the various portals have a login procedure (via token exchange) in a single step. The step locally works perfectly, while it continues to fail on AWS servers.

This issue has been analyzed through this ticket: [PN-9840: [E2E] Fix pipeline-PG](https://pagopa.atlassian.net/browse/PN-9840)

Changes Made:
- removed unused feature steps
- added new feature steps into PA,PF & PG for the login through `tokenExchange`

How to Test
Simply run any test and verify if all the described logic has been correctly implemented.
Also a run into AWS CodeBuild is necessary in order to test the full feature.
